src/test/scala/io/iohk/ethereum/db/storage/KeyValueStorageSuite.scala

```diff
@@ -0,0 +1,62 @@
+package io.iohk.ethereum.db.storage
+
+import io.iohk.ethereum.ObjectGenerators
+import io.iohk.ethereum.db.dataSource.{DataSource, EphemDataSource}
+import io.iohk.ethereum.rlp.RLPImplicits._
+import io.iohk.ethereum.rlp.{decode => rlpDecode, encode => rlpEncode}
+import org.scalacheck.Gen
+import org.scalatest.FunSuite
+import org.scalatest.prop.PropertyChecks
+
+import scala.util.Random
+
+class KeyValueStorageSuite extends FunSuite with PropertyChecks with ObjectGenerators{
```

Comment:
This spec doesn't test all public functions in a `KeyValueStorage`. Specs testing implementations of a `KeyValueStorage` are also not complete.